oasislmf/execution/bin.py

```diff
@@ -392,6 +399,44 @@ def prepare_run_inputs(analysis_settings, run_dir, ri=False):
         raise OasisException("Error preparing the model 'inputs' directory: {}".format(e))
 
 
+@oasis_log
+def set_footprint_set(setting_val, run_dir):
+    """
+    Create symbolic link to footprint file set that will be used for output
+    calculation.
+
+    :param setting_val: identifier for footprint set
+    :type setting_val: string
+
+    :param run_dir: model run directory
+    :type run_dir: string
+    """
+    format_to_class = {
+        'parquet': FootprintParquet, 'binZ': FootprintBinZ,
+        'bin': FootprintBin, 'csv': FootprintCsv
+    }
+    priorities = [format_to_class[fmt] for fmt in fp_format_priorities if fmt in format_to_class]
```

Comment:
This bit is already defined in oasislmf/pytools/getmodel/footprint.py
I'd say we should re-use the definition and avoid repetition.

Reply:
I agree that it would be better to avoid repetition. However, as the footprint format classes are defined in `oasislmf/pytools/getmodel/footprint.py`, we can't define `format_to_class` as a global variable before the declaration of these classes. I'm open to suggestions of how to make it so we only have to define it once.

Reply:
I'd then move def of `format_to_class` in oasislmf/pytools/getmodel/footprint.py after the classes are defined.

Reply:
We would also need to move the `Footprint` class to the end of the file too then.

Reply:
Well, actually I've just seen that `priorities` is already defined in the `load` method of `Footprint`. I think the code would benefit from a small refactoring, defining a priorities list and `format_to_class` mapping once in the `footprint.py` file, and using it in `Footprint` and everywhere else, including in `execution/bin.py`. For the sake of closing this PR quickly, probably it makes sense to leave it as it is, and remember/add an issue to refactor the footprint priorities. 
Right now we hard-code the priorities order and mapping three times in the code, which is bad for maintainability.

Reply:
Please see issue https://github.com/OasisLMF/OasisLMF/issues/1355.